src/devices/onesti.ts

```diff
@@ -26,8 +76,11 @@ const definitions: Definition[] = [
             device.save();
         },
         exposes: [e.lock(), e.battery(), e.sound_volume(),
-            e.lock_action_source_name(), e.lock_action_user(),
-            e.action(Array.from(Object.values(constants.easyCodeTouchActions))),
+            e.text('last_unlock_source', ea.STATE).withDescription('Last unlock source'),
```

Comment:
I missed this in my initial review, but this should be an `e.enum` with all the values from the lookup. (same for `last_lock_source`)